test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -5707,6 +5707,27 @@ def forward(self, input, other):
         y = torch.randint(10, (5, ))
         self.run_test(MatmulModel(), (x, y))
 
+    @skipIfUnsupportedMinOpsetVersion(9)  # MatMul long inputs is added in ONNX opset 9.
+    def test_dot(self):
+        class MatmulModel(torch.nn.Module):
+            def forward(self, input, other):
+                return torch.dot(input, other)
+
+        x = torch.randn(5, requires_grad=True)
```

Comment:
Just curious, why do we need to set requires_grad=True?

Reply:
could be potential use case. I mimicked the other Matmul test case.